pages/about.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8" />
+	<title>cathos/about</title>
+	<!-- Link to Style Sheet -->
+	<link href="styles/style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Eczar&display=swap" rel="stylesheet"> 
 </head>
+
 <body>
-  
+    <header>
+        <a id="site_title" href="index.html">
+            <h1>cathos</h1>
+        </a>
+        <nav>
+            <a href="about.html">about</a>
+            <a href="portfolio.html">portfolio</a>
+            <a href="contact.html">contact</a>
+            <a href="https://github.com/cathos">code</a>
```

Comment:
I would actually link to this to a page with links to particular projects. I would like to see specific projects you chose and the frameworks/languages you used to create those projects. A lot of hiring folks are not going to spend time looking through all of your repos on GitHub but skim through links you've provided and see what languages/frameworks you know.